terraform/applications/careers/static.tf

```diff
@@ -0,0 +1,189 @@
+# all the relevant infra for the static site
+resource "aws_s3_bucket" "logs" {
+  bucket = "mozilla-careers-logs-783633885093"
+  acl    = "log-delivery-write"
+}
+
+resource "aws_s3_bucket" "mozilla-careers" {
+  bucket = local.bucket_name
+  acl    = "log-delivery-write"
+
+  force_destroy = true
+
+  hosted_zone_id = local.s3_zone
+  logging {
+    target_bucket = aws_s3_bucket.logs.bucket
+    target_prefix = "stage-logs/"
+  }
+
+  website {
+    index_document = "index.html"
+  }
+
+  website_domain   = local.s3_website_domain
+  website_endpoint = local.s3_website_endpoint
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Id": "careers policy",
+  "Statement": [
+    {
+      "Sid": "careersAllowListBucket",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "s3:ListBucket",
+      "Resource": "arn:aws:s3:::${local.bucket_name}"
+    },
+    {
+      "Sid": "careersAllowIndexDotHTML",
```

Comment:
I imagine we're carrying this forward from a past environment, but I noted the dissonance between the ACL name and the actual allow wildcard of just "*".